packages/nimble-components/build/generate-workers/source/tests/render-worker.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { RenderWorker } from '../render-worker';
+
+describe('RenderWorker', () => {
+    let worker: RenderWorker;
+
+    beforeEach(() => {
+        worker = new RenderWorker();
```

Comment:
Instead of testing this directly, I think a more representative test would be to test this in process via MessageChannel.

See these comlink tests:

https://github.com/GoogleChromeLabs/comlink/blob/dffe9050f63b1b39f30213adeb1dd4b9ed7d2594/tests/same_window.comlink.test.js#L69-L75

https://github.com/GoogleChromeLabs/comlink/blob/dffe9050f63b1b39f30213adeb1dd4b9ed7d2594/tests/same_window.comlink.test.js#L195-L200

Reply:
You need to give more feedback if not following the comment when resolving if you don't want to get stuck in review forever.

I think your unit tests here should be with MessageChannel (that's why I commented here) and your integration tests with WebWorker (now there are none). 

Why was that suggestion ignored?

Reply:
Now they should be fine (I hope) 
![image](https://github.com/ni/nimble/assets/110180309/e0c1b934-39e0-498b-ad09-9f12a9043060)

Reply:
Hope it's fine now
![image](https://github.com/ni/nimble/assets/110180309/c0ce515a-92c0-4beb-a97c-3de1a737223a)